verification/curator-service/ui/cypress/integration/components/LinelistTableTest.spec.ts

```diff
@@ -15,12 +15,14 @@ describe('Linelist table', function () {
             notes: 'some notes',
             sourceUrl: 'www.example.com',
             methodOfConfirmation: 'PCR test',
+            curator: 'test@bar.com',
         });
         cy.visit('/cases');
         cy.contains('PCR test');
         cy.contains('some notes');
         cy.contains('France');
         cy.contains('www.example.com');
+        cy.contains('test@bar.com');
```

Comment:
Could you also add a test in the cypress NewCaseFormTest that the logged in user shows up on the case added?

Reply:
Sure, done.